Library/Services/INfieldSurveyInvitationTemplatesService.cs

```diff
@@ -25,8 +25,23 @@ namespace Nfield.Services
     public interface INfieldSurveyInvitationTemplatesService
     {
         /// <summary>
+        /// Add an invitation template for the specific survey
+        /// </summary>
+        Task AddAsync(string surveyId, InvitationTemplateModel invitationTemplate);
+
+        /// <summary>
         /// Get the invitation templates for the specified survey
         /// </summary>
         Task<IEnumerable<InvitationTemplateModel>> GetAsync(string surveyId);
+
+        /// <summary>
+        /// Update an invitation template for the specific survey
+        /// </summary>
+        Task UpdateAsync(string surveyId, InvitationTemplateModel invitationTemplate);
+
+        /// <summary>
+        /// Delete an invitation template for the specific survey
+        /// </summary>
+        Task<bool> DeleteAsync(string surveyId, string templateId);
```

Comment:
The request class for deleting an invitation has templateId as int: I changed the type of the parameter for the controller method as well. Any thoughts?